network/p2p/p2p_pubsub.go

```diff
@@ -226,16 +228,26 @@ func (n *p2pNetwork) handlePubsubMessages(logger *zap.Logger) func(ctx context.C
 		if msg == nil {
 			return nil
 		}
+		// TODO: check the logic
 
-		var decodedMsg *queue.DecodedSSVMessage
-		if msg.ValidatorData != nil {
-			m, ok := msg.ValidatorData.(*queue.DecodedSSVMessage)
-			if ok {
-				decodedMsg = m
-			}
+		// var decodedMsg *queue.DecodedSSVMessage
+		// if msg.ValidatorData != nil {
+		// 	m, ok := msg.ValidatorData.(*queue.DecodedSSVMessage)
+		// 	if ok {
+		// 		decodedMsg = m
+		// 	}
+		// }
+		// if decodedMsg == nil {
+		// 	return errors.New("message was not decoded")
+		// }
```

Comment:
@MatusKysel @pavelkrolevets , what do you mean? is this tested? please test on stage and remove that part if its checked and tested

Reply:
@y0sher brought it back. https://github.com/ssvlabs/ssv/pull/1442#discussion_r1677719930.